components/kyma-environment-broker/internal/storage/ext.go

```diff
@@ -8,7 +8,9 @@ import (
 
 type Instances interface {
 	FindAllJoinedWithOperations(prct ...predicate.Predicate) ([]internal.InstanceWithOperation, error)
+	FindAllInstancesForRuntimes(runtimeIdList []string) ([]internal.Instance, error)
 	GetByID(instanceID string) (*internal.Instance, error)
+	GetByRuntimeID(runtimeID string) (*internal.Instance, error)
```

Comment:
`GetByRuntimeID` is used by `getInstanceIDForShoots` which is not used anywhere.